django/contrib/postgres/indexes.py

```diff
@@ -11,11 +11,11 @@ class BrinIndex(Index):
     # applicable.
     max_name_length = 31
 
-    def __init__(self, fields=[], name=None, pages_per_range=None):
+    def __init__(self, fields=[], name=None, db_tablespace=None, pages_per_range=None):
```

Comment:
Could you move the `db_tablespace` attribute after `pages_per_range` please, just in case people pass that attribute as positional and not named arg. Also, thoughts about require `db_tablespace` to be a named argument: `def __init__(self, ..., pages_per_range=None, *, db_tablespace=None)`